spec/models/translation_override_spec.rb

```diff
@@ -7,6 +7,7 @@
     describe '#value' do
       before do
         I18n.backend.store_translations(I18n.locale, some_key: '%{first} %{second}')
+        I18n.backend.store_translations(:en, something: { one: '%{first} %{second}', other: '%{first} %{second}' })
```

Comment:
I suggest using different values for `one` and `other`. That way your test can ensure that the interpolation keys from `other` are used.